test/helpers/db.js

```diff
@@ -9,18 +9,10 @@ const Notification = require('../../src/models/db/notification').Notification
 const Fulfillment = require('../../src/models/db/conditionFulfillment').ConditionFulfillment
 
 exports.init = function * () {
+  yield knex.migrate.rollback(knexConfig).then()
```

Comment:
It definitely runs very slowly. Not sure how to get it to ignore indices, but would be worth looking into.